src/assets/styles/pages/_blog.scss

```diff
@@ -212,3 +212,22 @@
     font-size: 24px;
   }
 }
+
+blockquote {
+  margin: 0 0 10px;
+  padding: 63px 106px 63px 100px;
+  border-left: 8px solid #027FFF;
+  border-radius: 6px;
+  background-color: #f7f8fa;
+  color: #183028;
+  font-size: 20px;
+  line-height: 1.8em;
+  font-weight: 500;
+  margin-top: 48px;
+  margin-bottom: 48px;
+}
```

Comment:
1. Fabrizio, where do you take these color styles? Did someone provide you?
First, lets have all colors with variables if it's completely different color from our existing ones. If no, let's use the ones that we have (similar colors).
We have color variables in _variables.scss files
2. instead of 
margin-top: 48px;
  margin-bottom: 48px;
please use one line: margin: 48px 0;